src/ui/components/ShowMoreCard/ShowMoreCard.scss

```diff
@@ -2,7 +2,9 @@
 
 .ShowMoreCard-contents {
   &::after {
-    background: linear-gradient(transparent, $base-color);
+    // rgba(255,255,255,0) instead of transparent prevents
+    // grey line in Safari see #2865
```

Comment:
Can you link to the actual issue URL rather than the number? It makes it easier to get to and more obvious what that is... we know to search a GitHub repo for issues but a newbie might not. 😀